docs/config.qmd

```diff
@@ -405,6 +405,7 @@ lr_div_factor: # Learning rate div factor
 # - adamw_torch_fused
 # - adamw_torch_xla
 # - adamw_apex_fused
+# - adopt_adamw
```

Comment:
we should probably hint in the documentation that this is only for PyTorch version >= 2.5.1